src/devices/Pn532/RfConfiguration/MaxRetriesMode.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pn532.RfConfiguration
+{
+    /// <summary>
+    /// The parameters MxRtyATR, MxRtyPSL and MxRtyPassiveActivation define the
+    /// number of retries that the PN532 will use in case of the following processes
+    /// </summary>
+    public class MaxRetriesMode
+    {
+        /// <summary>
+        /// MxRtyATR is a byte containing the number of times that the PN532 will retry to
+        /// send the ATR_REQ in case of incorrect reception of the ATR_RES(or no
+        /// reception at all - timeout).
+        /// For active mode, value 0xFF means to try eternally, 0x00 means only once(no
+        /// retry, only one try). The default value of this parameter is 0xFF (infinitely).
+        /// For passive mode, the value is always overruled with 0x02 (two retries). 
+        /// </summary>
+        public byte MxRtyATR { get; set; } = 0x02;
```

Comment:
since this is public the names here should be expanded (i.e. MaxRetryATR - I don't know what ATR stands for but that ideally should be expanded too)